integration-tests/utils/integration-helpers.js

```diff
@@ -200,8 +200,8 @@ const clickNavChildAndLoad = async (page, tab) => {
 }
 
 const viewWorkspaceDashboard = async (page, token, workspaceName) => {
-  await click(page, clickable({ textContains: 'View Workspaces' }))
   await signIntoTerra(page, { token })
+  await click(page, clickable({ textContains: 'View Workspaces' }))
```

Comment:
Call `signIntoTerra()` first to handle unexpected NPS survey popup.

Reply:
You're too fast! 🙂 I was still typing comments while you were reviewing it.

Reply:
Ah right, you are doing the dismissal within signIntoTerra. Can you add a comment in the code? I think the original idea was that "View Workspaces" was sort of an entryway into logging into Terra, so people might be tempted to swap the order without a comment.

Reply:
Good idea. Added comment: `// Sign in to handle unexpected NPS survey popup and Loading Terra... spinner`